src/js/15-cloud-api-feedback.js

```diff
@@ -0,0 +1,72 @@
+(function () {
```

Comment:
this would get injected into every page. If we do it this way, we should fail fast and make sure it's running on the right page.

Reply:
I tested on the preview and looks good but would be a good idea to merge after https://github.com/redpanda-data/docs/pull/283 is merged too